src/util/magic.h

```diff
@@ -13,4 +13,7 @@ const std::size_t STRING_REFINEMENT_MAX_CHAR_WIDTH = 16;
 // Limit the size of strings in traces to 64M chars to avoid memout
 const std::size_t MAX_CONCRETE_STRING_SIZE = 1 << 26;
 
+// The top end of the range of integers for which dstrings are precomputed
+constexpr std::size_t DSTRING_NUMBERS_MAX = 64;
```

Comment:
Is there any value in using `constexpr` over `const`?

Reply:
@kroening ?

Reply:
Only that you can then could produce `constexpr` dstrings for numbers up to that value from a `constexpr` integer.